contracts/account-history-contract/src/entry_point/execute.rs

```diff
@@ -47,6 +50,14 @@ pub fn execute(
             if let Some(update_account_enabled) = update_account_enabled {
                 UPDATE_ACCOUNT_ENABLED.save(deps.storage, &update_account_enabled)?;
             }
+
+            if let Some(delete_old_data_enabled) = delete_old_data_enabled {
+                DELETE_OLD_DATA_ENABLED.save(deps.storage, &delete_old_data_enabled)?;
+            }
+
+            if let Some(delete_epoch) = delete_epoch {
+                DELETE_EPOCH.save(deps.storage, &delete_epoch)?;
+            }
```

Comment:
@amityadav0 can you also update this query endpoint https://github.com/elys-network/bindings/blob/main/contracts/account-history-contract/src/action/query/params.rs#L11 in order to query those two new params in the response thanks

Reply:
added